src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -4,27 +4,30 @@ class Webui::Users::NotificationsController < Webui::WebuiController
 
   # TODO: Remove this when we'll refactor kerberos_auth
   before_action :kerberos_auth
-  before_action :check_param_type, :check_param_project, only: :index
-
+  before_action :set_current_user, only: %i[index update]
   after_action :verify_policy_scoped
 
   def index
+    @selected_filter = params.permit!.to_h.except(*%w[controller action])
```

Comment:
Erm... why do you mark *every* parameter as permitted? This sounds like a huge trap for everything that comes after this PR. You know which parameters might come...

Reply:
You can specify that the nested keys just contain a hash.  So permit(project: {}, group: {}), and it should take "a hash of anything" at that point, to my understanding.

Reply:
I couldn't make that work before, but looks like now it's kinda working...